dataland-frontend/src/components/pages/CompanyCockpitPage.vue

```diff
@@ -1,9 +1,11 @@
 <template>
   <TheHeader v-if="!useMobileView" />
   <TheContent class="paper-section flex">
-    <CompanyInfoSheet :company-id="companyId" />
+    <CompanyInfoSheet :company-id="companyId" @fetched-company-information="getCompanyName" />
```

Comment:
Is this getCompanyName necessaryt to get the name of the company for the dialog?

Reply:
yes kind of, but also no.. 
maybe it is worth a little bit of refactoring. to avoid this. Let's have a quick call about my idea.

Reply:
Actually, it is not necessary anymore. And refactoring is not necessary as well. Thanks for the call :)

Reply:
removed it

Reply:
ok